specs/glossary.md

```diff
@@ -125,14 +125,41 @@ verified against the Merkle root.
 
 ## Chain Re-Organization
 
+Note: As of the current state of this PR, it is possible for "reorg" and "replacement" to be used as interchangable terms.
+
 [reorg]: glossary.md#chain-re-organization
 
-A re-organization, or re-org for short, is whenever the head of a blockchain (its last block) changes (as dictated by
-the [fork choice rule][fork-choice-rule]) to a block that is not a child of the previous head.
+A re-organization, or re-org for short, is whenever a Node on the blockchain changes (as dictated by
+the [fork choice rule][fork-choice-rule]) the head block (or multiple blocks) with other blocks.
+
+Reorgs are related to a reselection of branch on a node when there is some form of equivocation. Reorgs may not be seen
+by all nodes on the network, if those nodes picked the correct branch to begin with (for example, if they are syncing
+well after the reorg occured, only the canonical branch is available).
 
-L1 re-orgs can happen because of network conditions or attacks. L2 re-orgs are a consequence of L1 re-orgs, mediated via
+The following are types of reorgs which can be experienced by a node:
```

Comment:
These examples may not be comprehensive, and I'm not sure we would want them to be. Opinions?